src/site/_includes/macros/breadcrumbs.njk

```diff
@@ -0,0 +1,44 @@
+{#
+  @typedef {[string: string]} crumbs - The titles and urls of breadcrumbs to display.
```

Comment:
I thought this is from typescript linter. Copied from other macros in /macros folder

Reply:
Yup, of course. It's just that typedef means something very different - it should be param in both places.

Reply:
Well I think the idea was that this comment describes what object the macro is outputting in html / styling, and was not intended as function jsdoc. In such case, @typedef is reasonable and more universal - it describes a type present in the macro (in this case passed as a param, but that's not always the case, it can come from context in other macros).

How about I make you both happy and add both :)